From 6f001ca09fe5bb1aca85a3f532489b3aa412cd2a Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Fri, 2 Jun 2017 18:28:10 +0200
Subject: [PATCH 10/33] Don't try to use debug version of D-Bus library

Required for a debug build of Qt because mingw-w64-dbus
does not contain debug version

Change-Id: Ic34e1025fda55f9659e065f5bbe9d51f55420adb
---
 configure.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.json b/configure.json
index 17446b6c21..5b37371a7b 100644
--- a/configure.json
+++ b/configure.json
@@ -182,7 +182,7 @@
                 {
                     "libs": "",
                     "builds": {
-                        "debug": "-ldbus-1d",
+                        "debug": "-ldbus-1",
                         "release": "-ldbus-1"
                     },
                     "condition": "config.win32 && features.shared"
-- 
2.20.1

